src/selection_criteria.rs

```diff
@@ -131,11 +133,22 @@ impl<'de> Deserialize<'de> for ReadPreference {
 
         match preference.mode.as_str() {
             "Primary" => Ok(ReadPreference::Primary),
-            "Secondary" => Ok(ReadPreference::Secondary { options: preference.options }),
-            "PrimaryPreferred" => Ok(ReadPreference::PrimaryPreferred { options: preference.options }),
-            "SecondaryPreferred" => Ok(ReadPreference::SecondaryPreferred { options: preference.options }),
-            "Nearest" => Ok(ReadPreference::Nearest { options: preference.options }),
-            other => Err(D::Error::custom(format!("Unknown read preference mode: {}", other))),
+            "Secondary" => Ok(ReadPreference::Secondary {
```

Comment:
I assume rustfmt did this? I've been running into a few cases lately where it has reformatted code that I didn't touch but was in the same file I was working in. I guess that's the cost of sticking with the nightly version.